Makefile

```diff
@@ -106,3 +106,12 @@ check-headers:
 check-tidy:
 	go mod tidy
 	cd pkg/flowkit; go mod tidy
+
+.PHONY: check-env
+check-env:
+ifndef MIXPANEL_SERVICE_ACCOUNT_SECRET
+	$(error MIXPANEL_SERVICE_ACCOUNT_SECRET is undefined)
+endif
+ifndef MIXPANEL_PROJECT_TOKEN
+	$(error MIXPANEL_PROJECT_TOKEN is undefined)
+endif
```

Comment:
Test checks that MIXPANEL_SERVICE_ACCOUNT_SECRET and MIXPANEL_PROJECT_TOKEN will always be passed in. otherewise, ci will fail.

Reply:
This test is not necessary, since if this works then it will always work unless tokens are removed, which won't just happen by accident. The test that is necessary is making sure passing env variables work, so I would suggest adding a temporary action in the CI that checks if those variables are present in the Makefile, you can add an action that checks if values are present in makefile and then run that action as part of the CI, after the change validate that action has been run successfully on the CI and then you can remove it.

Reply:
I see. That's good test then. I would say you can remove this altogether from the CI since CI doesn't need to test this everytime. But good that we know it works this way.